backend/app.js

```diff
@@ -1,13 +1,23 @@
-const express = require('express');
-const bodyParser = require('body-parser');
-const questionRoutes = require('./routes/questionRoutes');
-const firebaseConfig = require('./config/firebaseConfig'); // Ensure Firebase is initialized
+const cors = require("cors"); 
+const express = require("express");
+const bodyParser = require("body-parser");
+const questionRoutes = require("./routes/questionRoutes");
+const firebaseConfig = require("./config/firebaseConfig"); // Ensure Firebase is initialized
 const app = express();
 
+// Allow requests from http://localhost:5173 with credentials
+app.use(cors({
+  origin: 'http://localhost:5173',
```

Comment:
Any particular reason why our frontend is hosted at 5173? Just curious 😆

Reply:
i just use what they give me and call it a day AHAHAHAHA